chainer/functions/normalization/group_normalization.py

```diff
@@ -83,14 +90,19 @@ def forward_cudnn(self, inputs):
         x, gamma, beta = inputs
         xp = cuda.cupy
 
+        interm_dtype = numpy.promote_types(x.dtype, gamma.dtype)
```

Comment:
This is not review.

I just fell like we should decide the name for this kind of variables of dtype used for mixed precision computation and provide the function to decide the dtype.

Reply:
I took the name from [BatchNormalization](https://github.com/chainer/chainer/blob/e0d92fe68a5285287d0b4e150abdca1c5b2a9198/chainer/functions/normalization/batch_normalization.py#L222).

Reply:
It seems that `interm_dtype` was introduced in #6484, so we should ask @hvy.

Reply:
If I remember correctly, the name comes from BN in ChainerX but it's of course open for discussion.

Reply:
I agree though that "highprec" as referred to in https://github.com/chainer/chainer/pull/7965#issuecomment-522332390 is more specific than "interm" and better in that sense. How about proceeding with this PR as is and discuss the naming in a different issue?

Reply:
It seems that the naming matter is not a problem of this PR. If the it is important, we should make another issue.